notebooks/checks/distribution/whole_dataset_drift.ipynb

```diff
@@ -0,0 +1,480 @@
+{
```

Comment:
<p>Title of the "drift bar" should has a redundant "Drift Score" in it (appears twice)</p><p> </p><p>And I think the design clarity / spacing should be improved a bit -</p><ol><li>Spacing between description and plot (maybe add new line before the drift bar?) and maybe add a title "Whole dataset total drift score" or something like that (right above the drift bar)</li><li>and also make somehow the distinction between the "total drift score" and the "main features contributing" a bit clearer? (new line? a physical line? something like that)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>